NEWS

```diff
@@ -5,6 +5,21 @@ testresources release notes
 IN DEVELOPMENT
 --------------
 
+2.0.2
+~~~~~
+
+CHANGES
+~~~~~~~
+
+* Skip testBasicSortTests in general (Kelledin)
```

Comment:
I think this pr is still open

Reply:
Are you suggesting that it be backed out of the main branch?

Reply:
https://github.com/testing-cabal/testresources/pull/13 is still open?